src/app/ApplicationTemplate.Shared.Views/Styles/Controls/TextBlock.xaml

```diff
@@ -133,13 +116,13 @@
 
 	<Style x:Key="Body3"
```

Comment:
For all the text styles, do we still need to re-define them in the app with V2?
I think we can now override the font family for material styles, so wouldn't that allow us to not redefine everything?

Reply:
I think you are right.

Reply:
... but do we still need to redefine all those styles?
Can't we just erase everything and override the font family in 1 place to automatically change all the material styles?